src/hooks/useProfitAndLossComparison/utils.ts

```diff
@@ -99,8 +100,8 @@ function preparePeriodsBodyForDateRange(dateRange: DateRange) {
   return {
     type: 'Comparison_Date_Ranges' as const,
     date_ranges: [{
-      start_date: dateRange.startDate.toISOString(),
-      end_date: dateRange.endDate.toISOString(),
+      start_date: toLocalDateString(dateRange.startDate),
+      end_date: toLocalDateString(dateRange.endDate),
```

Comment:
I would have _liked_ to do this formatting _right where the API call gets made_ but it would be messy here as
1. It's a `POST` request and we're taking in a `formattedBody` parameter, so it's weird to then do _extra_ formatting as we make the request
2. This is a polymorphic type that supports multiple request body formats, so you would only be remapping one of them, gross.

I decided to do the formatting in the `prepareFiltersBody` method, since this formatting does match the _intent_ of these methods and because it's the single place this body gets formatted now, so feels pretty safe that we have a single point of entry where we'll always do this formatting.